doc/source/cluster/config.rst

```diff
@@ -1,82 +1,286 @@
 .. _cluster-config:
 
-Configuring your Cluster
-========================
+Cluster Yaml Configuration Options
+==================================
```

Comment:
This is a nice page! However, I'm pretty worried about this going out of sync (which it most certainly will within 2 months). Instead, we should try to auto-gen this from the Ray configuration schema (ray-schema.json).